src/codeGenerators/orchestration/nodejs/toOrchestration.ts

```diff
@@ -145,18 +145,18 @@ export default function codeGenerator(node: any, options: any = {}): any {
       return `(${node.components.map(codeGenerator).join(` `)})`;
       return ` `;
 
-    case 'IfStatement': {
-      if(node.falseBody.length)
-      return `if (${codeGenerator(node.condition)}) {
-          ${node.trueBody.flatMap(codeGenerator).join('\n')}
-        } else {
-          ${node.falseBody.flatMap(codeGenerator).join('\n')}
-        }`
-        else
+      case 'IfStatement': {
+        if(node.falseBody.length)
         return `if (${codeGenerator(node.condition)}) {
-            ${node.trueBody.flatMap(codeGenerator).join('\n')}
```

Comment:
Can you confirm this please ? 
The orchestration is not generating falseBody statements. I tested with `If-Statement.zol`

Reply:
Ah it was for me because of the line in `orchestration-types` - though IIRC I only fully tested with internal calls, so I may have missed this